src/promptflow/promptflow/_trace/_start_trace.py

```diff
@@ -62,12 +62,16 @@ def _start_pfs_in_background(pfs_port) -> None:
     else:
         os.system(" ".join(["nohup"] + args + ["&"]))
 
-    wait_time = 0
+    wait_time = 10
```

Comment:
We  should improve the time here and add a internal API telemetry. We can do it in next PR